test/check-application

```diff
@@ -1326,6 +1326,13 @@ class TestApplication(testlib.MachineCase):
         # Set up command line
         b.set_input_text('#run-image-dialog-command', "sh -c 'for i in $(seq 20); do sleep 1; echo $i; done; sleep infinity'")
 
+        if auth:
+            # Set restart policy to 8 retries
+            b.set_val("#run-image-dialog-restart-policy", "on-failure")
+            b.set_input_text('#run-image-dialog-restart-retries', '3')
+        else:
+            b.wait_not_present("#run-image-dialog-restart-policy")
+
```

Comment:
Hmm, so enabling system service also works for user containers?

Reply:
It should also test that the service is actually started

Reply:
> Hmm, so enabling system service also works for user containers?

No? It should only be enabled for user containers.